exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -182,7 +184,10 @@ public Builder setServiceName(String serviceName) {
     }
 
     /**
-     * Sets the managed chanel to use when communicating with the backend. Required.
+     * Sets the managed chanel to use when communicating with the backend.
+     *
+     * <p>Should use only one of {@link #setChannel(ManagedChannel)} or {@link
```

Comment:
Does it make sense to validate this point in build?

Reply:
I updated this to match the OTLP exporter builders behavior and javadoc

Reply:
changed it again, no longer quite matches OTLP exporter behavior/javadoc, but i think a bit better (because of the default on endpoint, it's easier to check if channel was set, and if not fall back to endpoint). i will update my OTLP exporter PR to match